src/gc/gc.cpp

```diff
@@ -11958,6 +11958,7 @@ BOOL gc_heap::a_fit_segment_end_p (int gen_number,
         else
         {
             dprintf (2, ("can't grow segment, doing a full gc"));
+            dd_new_allocation (dynamic_data_of(gen_number)) += limit;
```

Comment:
@Maoni0 You're right. I added this line for debugging, and committed without delete this line.